Makefile

```diff
@@ -450,6 +453,30 @@ pillar-%: $(GOBUILDER) | $(DIST)
 clean:
 	rm -rf $(DIST) images/out pkg-deps.mk
 
+$(DOCKERFILE_FROM_CHECKER): $(DOCKERFILE_FROM_CHECKER)
+	make -C $(DOCKERFILE_FROM_CHECKER_DIR)
+
+.PHONY: check-docker-hashes-consistency
+check-docker-hashes-consistency: $(DOCKERFILE_FROM_CHECKER)
+	@echo "Checking Dockerfiles for inconsistencies"
+	$(DOCKERFILE_FROM_CHECKER) ./ \
+		-i ./eve-tools/bpftrace-compiler/Dockerfile \
+		-i ./eve-tools/bpftrace-compiler/root/Dockerfile \
```

Comment:
If we merge this PR after the original one, I believe we can avoid all these ignores?

Reply:
This PR is like the original one, but without bumping all the docker image tags.

If we merge this PR before the original one, we don't have the problem with running the Eden tests.

---

But actually I don't know what is @yash-zededa plan for this PR (see https://github.com/lf-edge/eve/pull/4334#issuecomment-2407687821 )

Reply:
We need the Eden tests only for the alpine version bumping. So, I guess, in this case, we can remove the workflow-related commits from the original PR, test it, merge it, and then - merge this one on top.